crates/viewer/re_space_view_dataframe/src/space_view_class.rs

```diff
@@ -110,46 +146,27 @@ for all entities, it is preferable to override the view-level visible time range
     ) -> Result<(), SpaceViewSystemExecutionError> {
         re_tracing::profile_function!();
 
-        let row_order = ViewProperty::from_archetype::<RangeTableOrder>(
+        let settings = ViewProperty::from_archetype::<DataframeSettings>(
             ctx.blueprint_db(),
             ctx.blueprint_query,
             query.space_view_id,
         );
-        let sort_key = row_order.component_or_fallback::<SortKey>(ctx, self, state)?;
-        let sort_order = row_order.component_or_fallback::<SortOrder>(ctx, self, state)?;
 
-        let mode = self.table_mode(query);
+        let mode = settings.component_or_fallback::<DataframeMode>(ctx, self, state)?;
+        let sort_key = settings.component_or_fallback::<SortKey>(ctx, self, state)?;
+        let sort_order = settings.component_or_fallback::<SortOrder>(ctx, self, state)?;
```

Comment:
It looks like these only apply to `TimeRange` mode, so why are we showing them in `LatestAt` mode? clicking them doesn't seem to do anything.

Reply:
Correct. I just changed it such that this part of the UI is disabled when the mode is not "Time Range", to make that clear.

Reply:
<img width="307" alt="image" src="https://github.com/user-attachments/assets/860b9a12-16f2-4487-8016-0067d58405fa">